packages/sandcastle/public/fonts/dejavu-sans-mono/DejaVu Fonts License.txt

```diff
@@ -0,0 +1,97 @@
+Fonts are (c) Bitstream (see below). DejaVu changes are in public domain.
```

Comment:
I'm a bit concerned about the license here. From what I can tell, this is not a standard license so we cannot guarantee  it's compatible with Apache 2.0.

Do we have an alternative with a standard license?